app/Http/Controllers/Api/PositionController.php

```diff
@@ -44,7 +44,24 @@ public function create()
      */
     public function store(Request $request)
     {
-        //
+        $user_id = Auth::user()->id;
+        $user_position = Position::where("user_id", $user_id)->get();
+
+        $position = new Position();
+        $position->user_id = null;
+        $position->department_id = $user_position[0]->department_id;
+        $position->grade_id = $request->pay_grade;
+        $position->name = $request->name;
+        $position->description = $request->description;
+        $position->hiring = 1;
+        $position->start_date = $request->start_date;
+        $position->end_date = $request->end_date;
+        $position->save();
+
+        return
+            [
+                'message' => 'succes'
```

Comment:
with the message, you can return id of the position created by user as well, something like this :
        return
            [
                'message' => 'succes', 'id' => $position->id,
            ];

Reply:
that will help you to useNavigate below and redirect

Reply:
Ok, I ammended it accordingly. Thanks a lot.